<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirusInfo</class>
 <widget class="QTabWidget" name="VirusInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget::pane {
		border-color: rgb(77,77,77);
		background-image: url(:/res/background.png);
		border-style: solid;
		border-width: 1px;
    	border-radius: 6px;
}
QTabBar::tab {
	font: 75 13pt &quot;Heiti SC&quot;;
	color: #808086;
	background-image:url(:/res/tabpressed.png);
}
QTabBar::tab:selected, QTabBar::tab:last:selected, QTabBar::tab:hover {
	color: #FFFFFF;
	background-image:url(:/res/tabdefault.png);
}
</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::North</enum>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="tab">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="title">
    <string>信息概览</string>
   </attribute>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>740</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-radius:20px;
border-width: 1px;
border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>传播途径</string>
   </attribute>
   <widget class="QWidget" name="SpreadTree" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>860</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>860</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-radius:20px;
border-width: 1px;
border-color: rgb(0, 0, 0);
background-image: url(:/res/windowbg.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>640</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;Heiti SC&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>传染性</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>650</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{background:rgba(171, 168, 162, 120);
border-style:solid;
border-radius:10px;
border-width: 1px;
border-color: rgba(155,155, 155, 169);} 
QProgressBar::chunk{border-radius:5px;background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(193, 0, 185, 255), stop:1 rgba(255, 94, 255, 255))};
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>640</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;Heiti SC&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>严重性</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>650</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{background:rgba(171, 168, 162, 120);
border-style:solid;
border-radius:10px;
border-width: 1px;
border-color: rgba(155,155, 155, 169);} 
QProgressBar::chunk{border-radius:5px;background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 121, 0, 255), stop:1 rgba(254, 255, 6, 255))};
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>640</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;Heiti SC&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>致命性</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>650</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{background:rgba(171, 168, 162, 120);
border-style:solid;
border-radius:10px;
border-width: 1px;
border-color: rgba(155,155, 155, 169);} 
QProgressBar::chunk{border-radius:5px;background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(49, 0, 139, 255), stop:1 rgba(190, 60, 255, 255))};
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>5</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Monaco&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Name of Virus</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>420</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>DNA</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>142</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Outline</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>130</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 24pt &quot;SimHei&quot;;</string>
     </property>
     <property name="text">
      <string>消耗点数以升级?</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="virus1">
    <property name="geometry">
     <rect>
      <x>929</x>
      <y>45</y>
      <width>301</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
	border-radius:10px;
	border-width: 2px;
	border-color: rgba(19, 19, 19, 100);
font: 72pt &quot;SimHei&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>440</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>440</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>0</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_hiden1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>650</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk{border-radius:5px;background:rgba(255, 94, 255, 100)};
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_hiden2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>650</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk{border-radius:5px;background:rgba(255, 255, 6, 100)};
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_hiden3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>650</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk{border-radius:5px;background:rgba(190, 60, 255, 100)};
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="info_widget1" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>505</y>
      <width>840</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#info_widget1{background-image: url(:/res/infoframe.png);}

</string>
    </property>
    <widget class="QLabel" name="SpreadTree_info1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>181</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;SimHei&quot;;
color:rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>信息框</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="description1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 18pt &quot;Heiti SC&quot;;</string>
     </property>
     <property name="text">
      <string>鸡你太美，鸡你实在是太美，oh baby......</string>
     </property>
    </widget>
    <widget class="QLabel" name="capacity1">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>10</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 22pt &quot;Heiti SC&quot;;
color:rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>sdfjk
sdfjk
sfjk</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>20</width>
       <height>100</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_3">
   <attribute name="title">
    <string>发病症状</string>
   </attribute>
   <widget class="QProgressBar" name="progressBar_4">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>630</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>640</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>传染性</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>630</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>640</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>严重性</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>630</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>929</x>
      <y>45</y>
      <width>301</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
	border-radius:30px;
	border-width: 5px;
	border-color: rgba(19, 19, 19, 100);
font: 72pt &quot;SimHei&quot;;</string>
    </property>
    <property name="text">
     <string>GIF区</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>860</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-radius:20px;
border-width: 1px;
border-color: rgb(0, 0, 0);
background-image: url(:/res/windowbg.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Monaco&quot;;</string>
    </property>
    <property name="text">
     <string>Name of Virus</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>851</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;SimHei&quot;;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimHei'; font-size:36pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-weight:600;&quot;&gt;描述特性区&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>420</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>41</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>DNA</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>130</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;SimHei&quot;;</string>
     </property>
     <property name="text">
      <string>消耗点数以升级?</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>640</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>致命性</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_4">
   <attribute name="title">
    <string>特种能力</string>
   </attribute>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>420</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>41</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>DNA</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>130</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;SimHei&quot;;</string>
     </property>
     <property name="text">
      <string>消耗点数以升级?</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar_7">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>630</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>640</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>传染性</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>860</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-radius:20px;
border-width: 1px;
border-color: rgb(0, 0, 0);
background-image: url(:/res/windowbg.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Monaco&quot;;</string>
    </property>
    <property name="text">
     <string>Name of Virus</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>630</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>851</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;SimHei&quot;;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimHei'; font-size:36pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-weight:600;&quot;&gt;描述特性区&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>640</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>致命性</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>640</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>严重性</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_9">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>630</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>60</y>
      <width>350</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
